toonz/sources/CMakeLists.txt

```diff
@@ -137,22 +160,22 @@ if(WIN32)
         set(QT_LIB ${SDKROOT}/quicktime/QT73SDK/Libraries/QTMLClient.lib)
     endif()
 else()
-    find_library(GLUT_LIB GLUT)
-    find_library(GL_LIB OpenGL)
+    # Apple & Linux
+    find_library(GLUT_LIB glut)
+    find_library(GL_LIB GL)
     find_library(Z_LIB z)
 
     find_package(TIFF REQUIRED)
     set(TIFF_LIB ${TIFF_LIBRARY})
     # the libraries have not .pc nor preferred Find*.cmake. use custom modules.
-    find_package(PNG REQUIRED)
-    message("******* libpng:" ${PNG_LIBRARY})
-    set(PNG_LIB ${PNG_LIBRARY})
+    # find_package(PNG REQUIRED)
+    set(PNG_LIB ${SDKROOT}/libpng-1.6.21/.libs/libpng16.so)
```

Comment:
you should only fallback to this when `find_package` fails to find system `libpng>=1.6`